lib/ramble/ramble/expander.py

```diff
@@ -829,8 +831,8 @@ def _eval_comparisons(self, node):
         except KeyError:
             raise SyntaxError("Unsupported binary comparison operator")
 
-    def _eval_comp_in(self, node):
-        """Handle in nodes in the ast
+    def _eval_comp_in_or_not(self, node, reverse):
+        """Handle in and not in nodes in the ast
 
         Perform extraction of `<variable> in <experiment>` syntax.
         Raises an exception if the experiment does not exist.
```

Comment:
Assuming the reverse logic does not move into this method, can you update the doc string to describe the arguments?

Reply:
Moved the logic up.